.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I think it is better to add a 'test.md' file in `.gitignore` because we don't need it for next step according to the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md)